apis/Google.Cloud.BigQuery.V2/Google.Cloud.BigQuery.V2.Snippets/BigQuerySnippetFixture.cs

```diff
@@ -84,6 +84,10 @@ private string CreateGameDataset()
 
         internal string GenerateDatasetId() => DatasetPrefix + Guid.NewGuid().ToString().Replace('-', '_');
 
+        internal string GenerateTableId() => Guid.NewGuid().ToString().Replace("-", "_");
+
+        internal string GenerateStorageBucketName() => Guid.NewGuid().ToString().Replace("-", "_");
```

Comment:
This doesn't conform to how this code used to name storage buckets. Is this intentional?